### DISCUSSION TITLE
Release Notes CovPass and CovPassCheck Version 1.16

### DISCUSSION THREAD

**Discussion by alexcimander** [Digitaler-Impfnachweis/covpass-ios#106] [Announcements]
📢 ***Release Notes CovPass and CovPassCheck Version 1.16*** 📢   

**Main Features CovPass:**

- Neutral Start Screen: The start screen has been visually adjusted so that it is no longer immediately obvious what type of certificate it is. This protects users' personal data and promotes the correct use of the CovPassCheck app.

- 2/1 Recognition: The apps now will detect 2/1 vaccinations (also 3/1, 4/1 etc.) as booster-vaccinations within the CovPass. An adjustment for CovPassCheck will be made using the digital EU Business Rules so that the new scheme will be not only accept but lead to immediately valid certificates etc.

  
**Planned Main Features CovPassCheck:**
•	No new main features are included in this version. A requested adaption for 2G+ concepts etc. is planned for the release 1.17.  Please also check the new announcement “Information about the upcoming version 1.17” for further information.

  
**Bug-Fixes:**
- Auto-Focus: Android-Devices that had troubles with the auto-focus while scanning certificates should now be able to scan certificates properly
- Remove-Onboarding on Updates: For Android the onboarding process will not be shown after an update. However new data-privacy statements need to accept during first startup after an update.
- Reference to full vaccination: With the neutral screen feature the information about the date of completed vaccination on the start screen will disappear. With that we will no longer reference the date of booster as the date of full immunization.
- Update on the rule sets: For IOS up-to-date local rule sets will be stored. With that boosters will be immediately valid after first startup. No manual update of the rule-sets is required.
- Fix on Flip- and Fold-Device Crashes: Flip- and Fold-Devices will no longer crash during scan. With that certificates can now be stored with the latest version of the CovPass and validated with the latest version of the CovPassCheck. Further UI adjustments are planned for upcoming releases.
  
❗ **Attention:**  Please also check the new announcement “Information about the upcoming version 1.17” for further information about upcoming features and bugfixes. Besides the upcoming features we are planning to adapt the current EU Rules to the new requirements. These updates are mostly independent of our release schedules. We will inform you about further updates.

Thank you all for your feedback, ideas and critics!   

stay safe and best regards,

Team CovPass and CovPassCheck ✔️

**Comment by Ein-Tim** [comment]
Same as #94?
